### PR TITLE
Check for merge conflicts on each push event

### PR DESCRIPTION
**Pull request by lilithxxx**
Changed Oppiabot's behavior to check for merge conflicts on each 'push' event
## Checklist
- [ ] I have successfully deployed my own instance of Oppiabot.
  - You can find instructions for doing this [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
- [ ] I have manually tested all the changes made in this PR following the [manual tests matrix](https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix).

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ module.exports = (robot) => {
     }
   });
 
-  robot.on('schedule.repository', async context => {
+  robot.on('push', async context => {
```
Comment (apb7) [inline index.js]: First of all, this is completely wrong and will render the merge conflict check unusable. Did you go through the [postmortem of the CLA check](https://docs.google.com/document/d/11DJBTlFqjsWU4oSThWNJKwLsaULcVGMOZVC3psZkHLM/edit#) which we released pretty recently? This is exactly what went wrong then.
Secondly, we have also added a checklist to ensure that all changes are manually tested. It is the responsibility of the PR author to fill it. _No untested PRs will be accepted, even if it involves a minor change_.

**Comment by apb7** [comment]
Hi @lilithxxx, any updates on this PR? Thanks!

**Comment by lilithxxx** [comment]
@apb7 I will get back to this once my exam ends. Thanks!

**Comment by apb7** [comment]
Closing this PR as it has been superseded by #32. Thanks!

**Pull request closed without merging**